src/ERC6538Registry.sol

```diff
@@ -1,21 +1,65 @@
-// SPDX-License-Identifier: UNLICENSED
-pragma solidity 0.8.20;
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
 
 import {IERC6538Registry} from "./interfaces/IERC6538Registry.sol";
```

Comment:
It's not a part of this PR, but I think we should remove this import and remove the `is` statement in the contract definition. Ideally these contracts should have no imports whatsoever. We can still *provide* the interfaces for use by others, and we should probably add some tests to make sure they are lined up and stay that way, as the code changes, but we really want to cut these contracts to the bone given the way they're a part of each respective standard.

Reply:
Agreed with @apbendi. I like the approach we used in CreateX, where the interface and contract are separate and we have a [CI check](https://github.com/pcaversaccio/createx/blob/7ef532c5ef4d972dc7c44ea8fd7c8ff70d196c90/.github/workflows/test-createx.yml#L88-L92) to make sure they're in sync (you can probably do it without the `jd` dep if you wanted also)

Reply:
Is having
```
interface IERC1271 {
  /// @dev Should return whether the signature provided is valid for the provided data
  /// @param hash      Hash of the data to be signed
  /// @param signature Signature byte array associated with _data
  function isValidSignature(bytes32 hash, bytes memory signature)
    external
    view
    returns (bytes4 magicValue);
}
```
at the bottom of the `ERC6538Registry.sol` fine? It lets us do high level contract call to verify 1271 signatures.
`IERC1271(registrant).isValidSignature(dataHash, signature)
              == IERC1271.isValidSignature.selector`